### ISSUE TITLE
erlang 19: crashed with reason: call to undefined function gen:debug_options([{max_queue,1000}])

### ISSUE THREAD

**Issue by adamflott** [processone/ejabberd#1176]
> What version of ejabberd are you using?

ejabberd 16.04-1
erlang 19.0-1

> What operating system (version) are you using?

Arch Linux 
Linux 4.6.2-1-ARCH #1 SMP PREEMPT Wed Jun 8 08:40:59 CEST 2016 x86_64 GNU/Linux

> How did you install ejabberd (source, package, distribution)?

Arch Linux Package

> What did not work as expected? Are there error messages in the log? What
> was the unexpected behavior? What was the expected result?

erlang 19: crashed with reason:
```
2016-06-27 20:41:50.352 [error] <0.10053.0> CRASH REPORT
 Process <0.10053.0> with 0 neighbours crashed with reason:
 call to undefined function gen:debug_options([{max_queue,1000}])

2016-06-27 20:41:50 =CRASH REPORT====
crasher:
initial call: ejabberd_c2s:init/1
pid: <0.10053.0>
registered_name: []
exception error: {undef,[{gen,debug_options,[[{max_queue,1000}]],[]},
{p1_fsm,init_it,6,[{file,"src/p1_fsm.erl"},{line,373}]},
{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]}
ancestors: [<0.597.0>,ejabberd_listeners,ejabberd_sup,<0.63.0>]
messages: []
links: []
dictionary: []
trap_exit: false
status: running
heap_size: 610
stack_size: 27
reductions: 134
neighbours:
```

Relevant info in: https://bugs.archlinux.org/task/49870#comment148485

**Comment by mremond** [comment]
Erlang OTP R19 has been release after ejabberd 16.04 and is not compliant with ejabberd 16.04.

Compliance with OTP R19 will likely happen with ejabberd 16.07. For ejabberd 16.06, it was already frozen by the time Erlang OTP R19 was released.

**Comment by adamflott** [comment]
Thanks! I'll keep my copy of erlang pinned at 18 until 16.07 is released. Thanks for the great product!!

**Comment by prefiks** [comment]
This should be fixed in version 16b1d8541aca8fb3029e3f7ea58b927609d63427
